### PR TITLE
Remove libWeb3Tests

### PR DESCRIPTION
**Pull request by winsvega**
Moved to libweb3core repo https://github.com/ethereum/libweb3core/pull/10

### CODE REVIEW

**Comment by winsvega** [comment]
https://github.com/ethereum/libweb3core/pull/10 (move with history)

File: test/libweb3core/crypto.cpp [hunk 1]
```diff
@@ -0,0 +1,150 @@
+/*
+	This file is part of cpp-ethereum.
```
Comment (chfast) [inline test/libweb3core/crypto.cpp:2]: Where does this file come from?

File: test/libweb3core/crypto.cpp [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+		cnote << toHex(rlp);
+		cnote << t.sha3(eth::WithoutSignature);
+		rlp = t.rlp(eth::WithSignature);
+		cnote << RLP(rlp);
+		cnote << toHex(rlp);
+		cnote << t.sha3(eth::WithSignature);
+		assert(t.sender() == p.address());
+	}
+
+#if 0
```
Comment (chfast) [inline test/libweb3core/crypto.cpp]: Can we remove this disabled code?

File: test/libweb3core/crypto.cpp (hunk 1 above)
Comment (winsvega) [inline test/libweb3core/crypto.cpp:2]: this are the tests for libweb3core that have dependencies on libethereum. thats why it is in libethereum.

**Pull request merged**